go/client/wallet_api_handler.go

```diff
@@ -465,7 +465,7 @@ func (c *sendOptions) Check() error {
 	if len(c.Message) > libkb.MaxStellarPaymentNoteLength {
 		return ErrMessageTooLong
 	}
-	if len(c.MemoText) > 28 {
+	if len(c.MemoText) > libkb.MaxStellarPaymentPublicNoteLength {
```

Comment:
I tried to remove as many magic `28`s, `500`s, and `240`s in the Go code and replace them with constants for maintainability.